BedrockServer.cpp

```diff
@@ -1284,32 +1288,39 @@ void BedrockServer::_status(BedrockCommand& command) {
         response.content = SComposeJSONObject(content);
     }
 
-    else if (SIEquals(request.methodLine, STATUS_WHITELIST)) {
-        SAUTOLOCK(_parallelCommandMutex);
+    else if (SIEquals(request.methodLine, STATUS_BLACKLIST)) {
+        SAUTOLOCK(_blacklistedParallelCommandMutex);
 
         // Return the old list. We can check the list by not passing the "Commands" param.
         STable content;
-        content["oldCommandWhitelist"] = SComposeList(_parallelCommands);
+        content["oldCommandBlacklist"] = SComposeList(_blacklistedParallelCommands);
 
         // If the Commands param is set, parse it and update our value.
         if (request.isSet("Commands")) {
-            _parallelCommands.clear();
+            _blacklistedParallelCommands.clear();
             list<string> parallelCommands;
             SParseList(request["Commands"], parallelCommands);
             for (auto& command : parallelCommands) {
-                _parallelCommands.insert(command);
+                _blacklistedParallelCommands.insert(command);
             }
         }
         if (request.isSet("autoBlacklistConflictFraction")) {
             BedrockConflictMetrics::setFraction(SToFloat(request["autoBlacklistConflictFraction"]));
         }
 
         // Enable extra logging in the commit lock timer.
-        decltype(SQLite::g_commitLock)::enableExtraLogging.store(!_parallelCommands.empty());
+        decltype(SQLite::g_commitLock)::enableExtraLogging.store(!_blacklistedParallelCommands.empty());
```

Comment:
Can you describe what this line does?  I don't really follow it.

Reply:
`decltype(SQLite::g_commitLock)` Is a compile-time lookup of the declared type of `SQLite::g_commitLock`, which currently resolves to `SLockTimer<recursive_mutex>`. `enableExtraLogging ` is an`atomic<bool>` so it's set and read via `load` and `store`.

A simpler version without the template handling and thread-safety and stuff would look like:
`SLockTimer:: enableExtraLogging = !_blacklistedParallelCommands.empty()`

All that actually does though is add a couple warnings if certain locks take longer than 1 second to acquire, and we probably don't even need this anymore.